cartofacade/carto_facade.go

```diff
@@ -178,12 +183,63 @@ type CartoFacade struct {
 	requestChan     chan Request
 }
 
-// RequestInterface defines the functionality of a WorkItem.
+// RequestInterface defines the functionality of a Request.
 // It should not be used outside of this package but needs to be public for testing purposes.
 type RequestInterface interface {
 	doWork(q *CartoFacade) (interface{}, error)
 }
 
+// Interface defines the functionality of a CartoFacade instance.
+// It should not be used outside of this package but needs to be public for testing purposes.
+type Interface interface {
+	request(
+		ctxParent context.Context,
+		requestType RequestType,
+		inputs map[RequestParamType]interface{}, timeout time.Duration,
+	) (interface{}, error)
+	start(
+		ctx context.Context,
+		activeBackgroundWorkers *sync.WaitGroup,
+	)
+
+	Initialize(
+		ctx context.Context,
+		timeout time.Duration,
+		activeBackgroundWorkers *sync.WaitGroup,
+	) error
+	Start(
```

Comment:
Can we distinguish this name from `start` somehow?

Reply:
done